components/brave_vpn/BUILD.gn

```diff
@@ -42,6 +42,11 @@ static_library("brave_vpn") {
 
   libs = []
 
+  sources += [
```

Comment:
nit: I think we don't need to add separately. Adding to above `source` is better?